graphql-java-servlet/src/main/java/graphql/kickstart/servlet/osgi/GraphQLMutationProvider.java

```diff
@@ -3,11 +3,7 @@
 import graphql.schema.GraphQLFieldDefinition;
 import java.util.Collection;
 
-public interface GraphQLMutationProvider extends GraphQLFieldProvider {
+public interface GraphQLMutationProvider extends GraphQLProvider {
```

Comment:
I would not touch so easily these interfaces since they are the contract to the outside and I think there are projects that could rely on them.

Same thing for the deletion of the GraphQLFieldProvider and the update of the GraphQlSubscriptionProvider